exporter/otlphttpexporter/otlp.go

```diff
@@ -0,0 +1,123 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otlphttpexporter
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/internal"
+	otlplogs "go.opentelemetry.io/collector/internal/data/opentelemetry-proto-gen/collector/logs/v1"
+	otlpmetrics "go.opentelemetry.io/collector/internal/data/opentelemetry-proto-gen/collector/metrics/v1"
+	otlptrace "go.opentelemetry.io/collector/internal/data/opentelemetry-proto-gen/collector/trace/v1"
+)
+
+type exporterImp struct {
+	// Input configuration.
+	config *Config
+	w      sender
+}
+
+type sender interface {
+	exportTrace(ctx context.Context, request *otlptrace.ExportTraceServiceRequest) error
+	exportMetrics(ctx context.Context, request *otlpmetrics.ExportMetricsServiceRequest) error
+	exportLogs(ctx context.Context, request *otlplogs.ExportLogsServiceRequest) error
+	stop() error
+}
```

Comment:
This interface is no longer required, was added with the idea that both grpc and http will be in the same exporter. Needs to be removed from grpc as well.

Reply:
Created the PR to remove from grpc https://github.com/open-telemetry/opentelemetry-collector/pull/1933